# Relevant research papers
- What did authors try to accomplish?
- What were the key elements of the approach?
- What can you use yourself?
- What other references do you want to follow?

## Paper 1
### A heterochromatin inducing protein differentially recognizes self versus foreign genomes
### https://doi.org/10.1371/journal.ppat.1009447
Heterochromatin plays a crucial role in gene regulation by repressing gene expression. Changes in chromatin structure, including shifts between heterochromatin and euchromatin states, can impact the expression of immune-related genes. Understanding these chromatin changes may provide insights into how the immune system responds to viruses.

Heterochromatin can affect the accessibility of DNA to transcription factors and other regulatory proteins. Changes in chromatin structure may impact the binding of transcription factors involved in the immune response. Investigating these interactions could be relevant to understanding how the immune system recognizes and responds to viral genomes.

## Paper 2
### Did viruses evolve as a distinct supergroup from common ancestors of cells?
### https://doi.org/10.1093/gbe/evw175
- No, not enough evidence. This means there is no ancestral relation between human and virus genomes.

## Paper 3
### Virus-derived variation in diverse human genomes
### https://doi.org/10.1371/journal.pgen.1009324

## Paper 4
### Virus-like insertions with sequence signatures similar to those of endogenous nonretroviral RNA viruses in the human genome
### https://doi.org/10.1073/pnas.2010758118

## Paper 5
### A fast and efficient algorithm for DNA sequence similarity identification
### https://doi.org/10.1007/s40747-022-00846-y

## Paper 6
### Deep learning of immune cell differentiation
### https://doi.org/10.1073/pnas.2011795117

## Paper 7
### Conserved Footprints of APOBEC3G on Hypermutated Human Immunodeficiency Virus Type 1 and Human Endogenous Retrovirus HERV-K(HML2) Sequences
### https://doi.org/10.1128/JVI.00584-08
#### What did authors try to accomplish
Find, from virus genome including patient-derived hypermutated HIV-1 genomes, signatures of hypermutation, guanine-to-adenine (G-to-A) hypermutation that strongly correlates to the hypermutation induced in vitro by the human polynucleotide cytidine deaminase APOBEC3G (hA3G) which is an antiviral restriction factor that induces this G-to-A hypermutation in retroviruses. This is to demonstrate the involvement of this restriction factor, APOBEC3G (hA3G), in defense against retroviral attacks over millions of years of human evolution.

#### What were the key elements of the approach

#### What can you use yourself
It is the APOBEC3G protein that is the footprint of human immune system found in virus genome.
Outcome: I can find out what type of viruses have this protein found in their genome.

## Paper 8
### APOBEC3 Proteins in Viral Immunity
### https://doi.org/10.4049/jimmunol.1501504
Backs up paper 7

## Paper 9
### Machine learning research towards combating COVID-19: Virus detection, spread prevention, and medical assistance
### https://doi.org/10.1016/j.jbi.2021.103751

## Paper 10
### DeepTCR is a deep learning framework for revealing sequence concepts within T-cell repertoires
### https://doi.org/10.1038/s41467-021-21879-w

## Paper 11
### Covid-19 and Artificial Intelligence: Genome sequencing, drug development and vaccine discovery
### https://doi.org/10.1016/j.jiph.2022.01.011

## Paper 12
### Prediction of human-virus protein-protein interactions through a sequence embedding-based machine learning method
### https://doi.org/10.1016/j.csbj.2019.12.005

## Paper 13
### A machine learning approach for viral genome classification
### https://doi.org/10.1186/s12859-017-1602-3

## Paper 14
### Prospective Artificial Intelligence to Dissect the Dengue Immune Response and Discover Therapeutics
### https://doi.org/10.3389/fimmu.2021.574411

## Paper 15
### Machine-learning-based identification of factors that influence molecular virus-host interactions
### https://doi.org/10.5525/gla.thesis.82931
"Due to a dependency on living entities, viruses have to rapidly exploit host systems for their replication and persistence. This promotes viruses to establish intimate molecular relationships with their hosts, a coevolutionary relationship that constantly shapes their genome through the need to maintain virus-host molecular interactions."

## Paper 16
### Predicting host taxonomic information from viral genomes: A comparison of feature representations
### https://doi.org/10.1371/journal.pcbi.1007894

## Paper 17
### Identifying and prioritizing potential human-infecting viruses from their genome sequences
### https://doi.org/10.1371/journal.pbio.3001390

## Paper 18
### Human immune system variation
### https://doi.org/10.1038/nri.2016.125

## Paper 19
### Evolution of Immune Systems From Viruses and Transposable Elements
### https://doi.org/10.3389/fmicb.2019.00051

## Paper 20
### Machine Learning for detection of viral sequences in human metagenomic datasets
### https://doi.org/10.1186/s12859-018-2340-x

## Paper 21
### ViraMiner: Deep learning on raw DNA sequences for identifying viral genomes in human samples
### https://doi.org/10.1371/journal.pone.0222271

## Paper 22
### Host Immune Response Driving SARS-CoV-2 Evolution
### https://doi.org/10.3390/v12101095
"In this work, we present a comprehensive genotyping analysis of existing SARS-CoV-2 mutations. We reveal that host immune response via APOBEC and ADAR gene editing gives rise to near 65% of recorded mutations."

## Paper 23
### Mechanisms of viral mutation
### https://doi.org/10.1007/s00018-016-2299-6

## Paper 24
### Evidence for host-dependent RNA editing in the transcriptome of SARS-CoV-2
### https://doi.org/10.1126/sciadv.abb5813

## Paper 25
### DNA Editing by APOBECs: A Genomic Preserver and Transformer
### https://doi.org/10.1016/j.tig.2015.10.005

## Paper 26
### Diverse functions for DNA and RNA editing in the immune system
### https://doi.org/10.4161/rna.7.2.11344
- "the AID/APOBEC cytidine deaminases (which catalyze the deamination of cytidine to uridine) "
- "other cytidine deaminases (APOBEC3) restrict retroviral infection by editing viral retrogenomes."
- "APOBEC3G  restricts  HIV  infection"
- "APOBEC3 family cytidine deaminases have diverse antiviral activities"
- "Aside from APOBEC3G, additional family members  APOBEC3B  and  APOBEC3F  can  also  restrict  HIV replication"

- "Apart from HIV, it seems that many retroviruses are some-what  susceptible  to  members  of  the  APOBEC3  subfamily; Murine   leukemia   virus   (MLV),   Equine   infectious   anemia virus  (EAIV),  foamy  retroviruses  and  even  hepatitis  B  virus (HBV)  can  be  restricted  by  different  APOBEC3  deaminases"
"APOBEC3A can dramatically inhibit replication of adeno-associated virus (A AV), a small single-stranded DNA parvovirus that replicates via the host cell polymerase machin-er y"

- "Endogenous  retroelements  are  suppressed  by  APOBEC3 family  cytidine  deaminases.  The  suppressive  activity  of  the APOBEC3  proteins  is  not  limited  to  exogenous  retroviruses; several family members have also been demonstrated to restrict a  variety  of  endogeneous  retroelements."
"Perhaps  retroelement  inhibition represents  an  “original”  function  of  the  APOBEC3  proteins; activity against exogenous lentiviruses may have been co-opted later  in  the  evolutionary  history  of  this  gene  family.  Indeed, the  apparent  selective  pressures  on  the  APOBEC3  genes  pre-date the emergence of modern primate lentiviruses by millions of  years.54,55 In addition, the evolutionary expansion of the APOBEC3 family correlates with a dramatic decrease of endog-enous  retroelement  activity  in  primate  genomes  as  compared to  rodents.65  Thus,  the  APOBEC3  proteins  may  represent  an ancient  defense  system  for  protecting  genome  integrity,  be  it from actively mobile endogenous retroelements or the primitive retroviruses from which they were derived."

## Paper 27
### Rapid evolution of primate antiviral enzyme APOBEC3G
### https://doi.org/10.1093/hmg/ddh183







